modules/ROOT/pages/nav-matrix.adoc

```diff
@@ -5,7 +5,7 @@ include::{partialsdir}/attributes.adoc[]
 [discrete]
 == Prerequisites
 
-* [x] You have xref:ROOT:index.adoc[set up AeroGear Mobile Services on OpenShift].
+* [x] You have set up xref:getting-started.adoc[AeroGear Mobile Services] on OpenShift.
 
 * [x] You have xref:registering-a-mobile-app.adoc[registered a mobile app] to create a {mobile-client}.
```

Comment:
@finp Should this be linking to the relevant anchor in the larger getting started document rather than a specific file. Not clear on how this is supposed to work...

Reply:
We should use an anointed url, in this case I'd suggest:

xref:getting-started.adoc#registering[registered a mobile app]